src/http/index.js

```diff
@@ -319,8 +319,19 @@ function formatKeyValueBySerializationOption(key, value, skipEncoding, serializa
     : serializationOption && serializationOption.allowReserved
       ? 'unsafe'
       : 'reserved';
-  const encodeFn = (v) => encodeDisallowedCharacters(v, { escape });
-  const encodeKeyFn = skipEncoding ? (k) => k : (k) => encodeDisallowedCharacters(k, { escape });
+
+  const encodeFn = (v) => {
```

Comment:
We use exported from https://github.com/swagger-api/swagger-js/pull/3446/files#diff-5f9165415636dea264548b1e96c83949a93c5a3b6ae9af3951cfb1fe63443c61R59